crypto/cryptlib.c

```diff
@@ -319,14 +319,9 @@ void OPENSSL_die(const char *message, const char *file, int line)
  * pointers to volatile by preserving the reads and writes through them.
  * The latter is not required by the Standard if the memory pointed to
  * is not volatile.
- * Pointers themselves are volatile in the function signature to work
- * around a subtle bug in gcc 4.6+ which causes writes through
- * pointers to volatile to not be emitted in some rare,
- * never needed in real life, pieces of code.
```

Comment:
I apologize for sounding picky, but I'd vote for removal of this comment completely. 1. is irrelevant and 2. is ... I don't know what is it, because I can't relate to it at all... But it would be appropriate to have some commentary, so that question is rather about replacement than removal. Should I try to put some words together?